client/src/app/components/AppDialogShell.vue

```diff
@@ -13,9 +13,8 @@
           div(class="dialog-footer-errors is-pulled-left")
             div(v-show="errors.length")
               span {{ errorDirections }} 
-              span(v-for="(error, index) in errors", class="error-color")
-                span(v-if="errors.length > 0 && index < errors.length - 1") {{ error }}, 
-                span(v-else) {{ error }}
+              span(v-for="(error, index) in errors.join(', ')", class="error-color")
```

Comment:
Nice yet doesn't array.join return a single string so there is no longer a need for the v-for.  Right?  :)

Reply:
@bryan-gilbert, true. Sorry for this. I have made another commit addressing this. Thank you for the heads-up!